ios/brave-ios/Sources/BraveWallet/Crypto/Portfolio/PortfolioView.swift

```diff
@@ -180,7 +180,7 @@ struct PortfolioAssetGroupHeaderView: View {
         if case .network(let networkInfo) = group.groupType {
           NetworkIconView(network: networkInfo, length: 32)
         } else if case .account(let accountInfo) = group.groupType {
-          Blockie(address: accountInfo.address)
+          Blockie(address: accountInfo.blockieSeed)
```

Comment:
Can we have some `Blockie` factory which is able to create an instance from just accountInfo?

Reply:
I think its fine to keep it only taking the String value since we are not only using it for `AccountInfo` but also for string type address from `TxInfo`, dapps' requests and even for custom network's icon based on network name. 
`blockieSeed` is added as en extension computed variable in `AccountInfo` which can easily check what is the correct seed value for this `AccountInfo`